### ISSUE TITLE
Client startup language error

### ISSUE THREAD

**Issue by alexgrist** [Nak2/StormFox#14]
When using the GitHub version without being subscribed to the workshop addon, StormFox will fail to load properly with the following error:

```
[StormFox] V 1.5.

[ERROR] lua/includes/extensions/string.lua:87: bad argument #1 to 'string_len' (string expected, got nil)
  1. string_len - [C]:-1
   2. Explode - lua/includes/extensions/string.lua:87
    3. LoadLangauge - addons/stormfox/lua/autorun/stormfox_autorun.lua:60
     4. unknown - addons/stormfox/lua/autorun/stormfox_autorun.lua:77
````

Likely due to lua files not being readable by file.Read if they're sent over via lua cache.

**Comment by PolSpock** [comment]
Same here with workshop version

**Comment by Nak2** [comment]
I've added this as a priority for v1.6.
Sorry for the slow response.

**Comment by nateabowman** [comment]
I can't get it to load up now, same error.

**Comment by Nak2** [comment]
Should be fixed in 1.6.
